main.py

```diff
@@ -18,26 +18,30 @@
 parser.add_argument('--preserve', '-p', metavar='P', type=bool, default=False,
                     help="whether to preserve previous character runs, " + 
                     "which is useful if you want to display more characters (default: False)")
+parser.add_argument('--template', '-t', metavar='T', type=int, help="teamplate", default=3)
+
 args = parser.parse_args()
 
 uid       = args.uid
 outputdir = args.outputdir
 imgdir    = outputdir if args.imgdir is None else args.imgdir
 lang      = args.lang
 preserve  = args.preserve
+template  = args.template
 
 if os.path.exists(outputdir):
     if not preserve:
         shutil.rmtree(outputdir)
 os.makedirs(outputdir, exist_ok=True)
 
 async def main():
-    async with honkaicard.MiHoMoCard(lang=lang) as hmhm:
+    async with honkaicard.MiHoMoCard(lang=lang, template = template, background = False) as hmhm:
```

Comment:
In my opinion, the background turn-off here is not necessary. Keeping the consistent design as a card object is better perhaps.

```suggestion
    async with honkaicard.MiHoMoCard(lang=lang, template = template) as hmhm:
```